integrations/tensorflow-serving/documentation.yaml

```diff
@@ -0,0 +1,115 @@
+exporter_type: included
+app_name_short: TF Serving
+app_name: TensorFlow Serving
+app_site_name: {{app_name_short}}
```

Comment:
I'll change this to `{{app_name}}` because the title of the linked page https://github.com/tensorflow/serving is "TensorFlow Serving"